pkg/executor/point_get.go

```diff
@@ -246,7 +246,14 @@ func (e *PointGetExecutor) Open(context.Context) error {
 // Close implements the Executor interface.
 func (e *PointGetExecutor) Close() error {
 	if e.stats != nil {
-		defer e.Ctx().GetSessionVars().StmtCtx.RuntimeStatsColl.RegisterStats(e.ID(), e.stats)
+		defer func() {
+			sc := e.Ctx().GetSessionVars().StmtCtx
+			sc.RuntimeStatsColl.RegisterStats(e.ID(), e.stats)
+			timeDetail := e.stats.SnapshotRuntimeStats.GetTimeDetail()
+			if timeDetail != nil {
+				sc.SyncExecDetails.MergeTikvCPUTime(timeDetail.ProcessTime)
```

Comment:
What is the difference between ProcessTime and TikVCPUTime? As I understand it, the TiKVCPUTime is the sum of ProcessTime of each cop/point_get task. Of cource, the ProcessTime for sql is also the sum of ProcessTime of each cop/point_get task.

Reply:
ProcessTime doesn't include point_get task's time now. Point_get's execution details are only stored in execution summary.